### PR TITLE
Add zachjs-sv2v package

### PR DESCRIPTION
**Pull request by tjurtsch**
The recipe is based on 'SymbiFlow/conda-packages' one.

### CODE REVIEW

File: zachjs-sv2v/condarc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```
[the remark this answers is not in the document]
Comment (tjurtsch) [reply zachjs-sv2v/condarc]: I removed it and it's clear that 'stack' Conda package is pulled from conda-forge channel
https://travis-ci.com/github/litex-hub/litex-conda-misc/jobs/428033336

[the remark this answers is not in the document]
Comment (PiotrZierhoffer) [reply zachjs-sv2v/condarc]: ok

[the remark this answers is not in the document]
Comment (tjurtsch) [reply zachjs-sv2v/condarc:2]: Yes, I re-added it

**Comment by tjurtsch** [comment]
I noticed that recipe from 'Symbiflow/conda-packages' is failing, although no changes to the recipe were made.
Last successful build: https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/427344144
First failing build: https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/428972925

I tracked down the cause of the issue. In successful builds, host's compiler was used (installed via "apt"). Failing builds use Conda-provided compiler and it clearly have problem with linking gmp library. Compiler issue was also described here: https://github.com/haskell/cabal/issues/5280#issuecomment-718818157

Conda build started to use Conda compilers because there was a change in "stack" (Haskell tool) recipe in conda-forge channel.
In version 2.5.1-0 there's no "requirements" section in meta.yaml.
In version 2.5.1-h57928b_1 (latest) gcc-linux-64 was added as a requirement.
Both versions are recognized as 2.5.1 by conda-build, so the latest one is pulled during build.

We will fix this recipe for now, by using host's compiler (not Conda) and non-Conda stack version.

**Comment by PiotrZierhoffer** [comment]
@mithro this is a serious issue that might explode when we start digging into it.

I suggest uploading the one that works (even if not compiled as it should be) not to get blocked by it.

File: zachjs-sv2v/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+set -x
+set -e
+
+STACK_BINARY="https://github.com/commercialhaskell/stack/releases/download/v2.5.1/stack-2.5.1-linux-x86_64-bin"
+
+# Pull Haskell stack binary
+# stack version from conda-forge forces us to use
```
Comment (PiotrZierhoffer) [inline zachjs-sv2v/build.sh:9]: can we just force an older version of stack?

Comment (tjurtsch) [reply zachjs-sv2v/build.sh]: Yes we can. Recipe updated.

**Pull request merged**